packages/sitecore-jss-nextjs/src/services/graphql-sitemap-service.ts

```diff
@@ -6,36 +6,71 @@ export const languageError = 'The list of languages cannot be empty';
 
 const languageEmptyError = 'The language must be a non-empty string';
 
-const defaultQuery = /* GraphQL */ `
-  query SitemapQuery(
-    $siteName: String!
-    $language: String!
-    $includedPaths: [String]
-    $excludedPaths: [String]
-    $pageSize: Int = 10
-    $after: String
-  ) {
-    site {
-      siteInfo(site: $siteName) {
-        routes(
-          language: $language
-          includedPaths: $includedPaths
-          excludedPaths: $excludedPaths
-          first: $pageSize
-          after: $after
-        ) {
-          total
-          pageInfo {
-            endCursor
-            hasNext
-          }
-          results {
-            path: routePath
+const defaultQuery = /* GraphQL site path query without personalization (works on sxp and xm cloud) */ `
+query DefaultSitemapQuery(
+  $siteName: String!,
+  $language: String!,
+  $includedPaths: String[],
+  $excludedPaths: String[],
+  $pageSize: Int = 10,
+  $after: String
+) {
+  site {
+    siteInfo(site: $siteName) {
+      routes(
+        language: $language
+        includedPaths: $includedPaths
+        excludedPaths: $excludedPaths
+        first: $pageSize
+        after: $after
+      ){
+        total
+        pageInfo {
+          endCursor
+          hasNext
+        }
+        results: routesResult{
+          path: routePath 
+        }
+      }
+    }
+  }
+} 
+`;
+
+const personalizeQuery = /* GraphQL site path query with personalization (xm cloud only)*/ `
```

Comment:
Can we avoid duplicating these queries? I think moving the query string to the `query()` getter method would allow us to combine these into a single string literal with a conditional on the personalize portion.